src/vivarium_csu_alzheimers/utilities.py

```diff
@@ -165,13 +165,13 @@ def get_lognorm_from_quantiles(
     return stats.lognorm(s=sigma, scale=median)
 
 
-def get_random_variable_draws(
```

Comment:
I copied these versions of the functions straight from mncnh. We really should refactor Artifact stuff into its own thing.